wagtail/core/migrations/0047_add_page_uuid.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.12 on 2020-06-21 09:04
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('wagtailcore', '0046_site_name_remove_null'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='page',
+            name='uuid',
+            field=models.UUIDField(editable=False, null=True),
```

Comment:
I noticed this isn't `unique=true`. So I'm wondering if this could be used by wagtail-localize to link translations of the same page?

Currently, this is done with a shared UUID: https://github.com/wagtail/wagtail-localize/blob/master/wagtail_localize/models.py#L97